### ISSUE TITLE
Duplication in XML

### ISSUE THREAD

**Issue by kaitlinnewson** [asmecher/jatsTemplate#2]
Hi @asmecher, more feedback from our metadata librarian:

There is a lot of duplication. We can handle this in the loader, but it would be ideal if we didn't have to and it was cleaned up on their side. This includes straight up duplication, e.g.:

<subj-group subj-group-type="heading">
<subject>Articles</subject>
</subj-group>
<subj-group subj-group-type="heading">
<subject>Articles</subject>
</subj-group>

As well as instances of the same information being presented in a slightly different way. E.g.:

<pub-date date-type="pub" iso-8601-date="2007-06-15" publication-format="epub">
<day>15</day>
<month>06</month>
<year>2007</year>
</pub-date>
<pub-date pub-type="epub">
<day>15</day>
<month>06</month>
<year>2007</year>
</pub-date>

**Comment by asmecher** [comment]
@kaitlinnewson, thanks for the feedback. You're using both the JATS Template and OAI JATS plugins, correct? And this output comes via the OAI interface? (Just trying to make sure I know the potential sources of the duplication you're seeing.)

**Comment by kaitlinnewson** [comment]
@asmecher Yes, we're using both plugins, then harvesting the XML via OAI. Let me know if you'd like more details.

**Comment by asmecher** [comment]
OK, thanks. Are these installed via the Plugin Gallery, or e.g. from Github? What version are you using for each of the two?

**Comment by kaitlinnewson** [comment]
Both are v. 1.0.0.0, installed via the plugin gallery. The OJS instance is v. 3.1.1.4.

**Comment by asmecher** [comment]
The `subj-group` duplication is resolved here: https://github.com/pkp/oaiJats/commit/c35e2b2624f6c4faa860c5f8e3c8d6f9b2b61471
